hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/dto/nomis/OffenderBooking.kt

```diff
@@ -49,6 +49,7 @@ data class OffenderBooking(
 private fun getDiffResult(booking: OffenderBooking, other: OffenderBooking): DiffResult<OffenderBooking> =
   DiffBuilder(booking, other, ToStringStyle.JSON_STYLE).apply {
     OffenderBooking::class.members
-      .filterNot { listOf("diff", "equals", "toString", "hashCode").contains(it.name) }
+      .filterNot { listOf("copy", "diff", "equals", "toString", "hashCode").contains(it.name) }
+      .filterNot { it.name.startsWith("component") }
```

Comment:
Don't want to diff on all the "special" methods provided by Kotlin because OffenderBooking is a data class.